engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -62,18 +90,25 @@ public class GroovyDeephavenSession extends AbstractScriptSession<GroovySnapshot
     private static final String DEFAULT_SCRIPT_PATH = Configuration.getInstance()
             .getStringWithDefault("GroovyDeephavenSession.defaultScriptPath", ".");
 
+    private static final boolean INCLUDE_DEFAULT_IMPORTS_IN_LOADED_GROOVY =
+            Configuration.getInstance()
+                    .getBooleanWithDefault("GroovyDeephavenSession.includeDefaultImportsInGroovyScripts", false);
+    private static final boolean INCLUDE_CONSOLE_IMPORTS_IN_LOADED_GROOVY =
+            INCLUDE_DEFAULT_IMPORTS_IN_LOADED_GROOVY && Configuration.getInstance()
```

Comment:
Perhaps - I wrote it this way under the assumption that a user who expects console imports to work in scripts would want that for the purpose of basically authoring notebooks (copied contents from the console), rather than "actually that script is missing a single import, so I'll run it manually."

That is, when would a user _not_ expect to need an import for `com.company.project.MyClass` if they had already used it once in the console, but _expect_ to write imports for TableTools etc (which are in the default imports list).

Reply:
Discussed with Colin off GH, which resulted in my request at https://github.com/deephaven/deephaven-core/pull/4596#discussion_r1361051744.